search_filter_instructions.txt

```diff
@@ -0,0 +1,59 @@
+Copy the code listed below and incorporate rate the three bodies in with the correct
+file.  HTML goes in index,  Javascript goes in JS file.
+We will need to change the HTML table to match the filters we choose as well as update
+the data with actual data relevent to our site.  The options listed below are just
+as an example.
+
+HTML
+
+Place this in the header.
+
+<script src="http://code.jquery.com/jquery-1.4.2.min.js"></script>
+
+
+Place this HTML in with the search feature code. As said, we will have to swap out the data for what is relevent to our project.
+
+<h1 id="header">DVD Collection</h1>
+
+<ul id="list">
+  <li><a href="#">Batman</a></li>
+  <li><a href="#">Ice Age</a></li>
+  <li><a href="#">Star Trek (2009)</a></li>
+  <li><a href="#">Tremors</a></li>
```

Comment:
This is a good start to our list. We should have much more content included on the page before we can merge though. Also, not sure that I recognize any of the poems on lines 19-22.